### DISCUSSION TITLE
HydroMT-wflow installation

### DISCUSSION THREAD

**Discussion by Starphire1** [Deltares/hydromt_wflow#215] [Q&A] [answered]
Hi, noting an issue accessing the url under "Install HydroMT-Wflow in a new environment" https://raw.githubusercontent.com/Deltares/hydromt_wflow/. Could this please be checked? Cheers.

**Comment by JoostBuitink** [comment]
This link is indeed outdated, and is fixed in [this PR](https://github.com/Deltares/hydromt_wflow/pull/211). This will be fixed in the coming days.

**Reply by JoostBuitink** [reply, marked as answer]
The docs are now fixed, see this page: https://deltares.github.io/hydromt_wflow/latest/getting_started/installation.html#install-hydromt-wflow-in-a-new-environment

**Reply by Starphire1** [reply]
Thank you. Testing the new install option now.
